engine/battle/ai/items.asm

```diff
@@ -725,10 +725,14 @@ AI_HealStatus:
 	xor a
 	ld [hl], a
 	ld [wEnemyMonStatus], a
-	; Bug: this should reset SUBSTATUS_NIGHTMARE too
-	; Uncomment the lines below to fix
+	; Bug: this should reset SUBSTATUS_NIGHTMARE
+	; Uncomment the former 2 lines below to fix
+	; This should also reset SUBSTATUS_CONFUSED
+	; Uncomment the latter 2 lines below to fix
```

Comment:
Why not just put each comment block before each code block?

Reply:
I can see your point there. It'd make more sense for the beginner the way you're thinking of.

Reply:
LGTM

Reply:
Fixed.